### ISSUE TITLE
DUP: FOI Request

### ISSUE THREAD

**Issue by Dianadec** [bcgov/parks-reso-public#463]
#### Description:

See Teams thread: https://teams.microsoft.com/l/message/19:7gg3pxzoL-hjZRHvr5ggkjxP9O8cah6SWqiYiz2CD_M1@thread.tacv2/1727985265657?tenantId=6fdb5200-3d0d-4a8a-b036-d3685e359adc&groupId=aadf267c-d6ab-4c8c-9cff-e574713575e6&parentMessageId=1727985265657&teamName=External%3A%20Team%20Osprey&channelName=General&createdTime=1727985265657

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by LindsayMacfarlane** [comment]
Hi @cameronpettit - Thanks for pulling this together! Double checking that we weren't able to pull data for the summer 2023 season because it isn't available?

**Comment by LindsayMacfarlane** [comment]
@cameronpettit - still awaiting feedback on the data. Did receive some clarification from Marshall re the 86% of days sold out at Joffre. Him and Sally manage a spreadsheet manually that takes into consideration closure periods. They estimate the sold out period at 96% from June to Aug 31.

**Comment by cameronpettit** [comment]
Oh darn I misread. I can see if we have data going back to 2023. I was under the impression it was only for this summer.

**Comment by cameronpettit** [comment]
@LindsayMacfarlane redid the information collection between `2023-06-01 to 2024-08-31` so now there are 244 files in this folder:

https://bcgov.sharepoint.com/:f:/r/teams/08548/Shared%20Documents/General/1.%20Project%20Documentation/2.%20DUP/FOI%20-%20October2024/[selloutTimes](https://bcgov.sharepoint.com/:f:/r/teams/08548/Shared%20Documents/General/1.%20Project%20Documentation/2.%20DUP/FOI%20-%20October2024/selloutTimes?csf=1&web=1&e=RkcEM9)?csf=1&web=1&e=RkcEM9

Some slight changes with the data for this new time range - we introduced the hold pass timer in about May 2024 which, as previously mentioned, adds a 7 minute buffer to the amount of time a pass is held for a user to fill out their personal information before a pass is returned to the availability pool. Before this was introduced, a pass was considered 'sold' when the transaction was completed so people who could enter their user information quickly had an advantage. With these two different methods, the average sellout time is skewed when looking at times before and after the hold pass timer was introduced. Before it was introduced, some of the faster sellout times are around 30 seconds. After it was introduced, some of the faster sellout times were around 7 minutes 30 seconds - but all the passes were likely in a state of 'holding' 30 seconds after opening. 

Also, Joffre Lakes halved its capacity in 2024, so 2023 sellout times are skewing the data to have longer durations to sellout on average.

Days that no reservations were recorded were not included in the data - Joffre Lakes was most likely closed on these days. 

```json
 "startDate": "2023-06-01",
 "endDate": "2024-08-31",
 "totalDays": 458,
 "daysWithData": 222,
 "totalDaysSoldOut": 200,
 "percentageOfDaysSoldOut": 90.09009009009009,
 "averageTimeOfSellout": "09:08:52",
 "averageDurationAsMinutes": 128.87578813813815,
 "fastestSelloutTime": 0.49633333333333335
```

**Comment by LindsayMacfarlane** [comment]
@cameronpettit @Dianadec - I'm going to consider this ticket closed. I will create a new ticket(s) when we get more information related to the request. Thanks for all your work pulling this information together Cam!
